tests/index.js

```diff
@@ -56,6 +56,10 @@ describe('<ContentLoader />:', () => {
       expect(wrapper.props().width).to.equal(200)
     })
 
+    it('`animate` is a boolean and used', () => {
+      expect(wrapper.props().animate).to.equal(true)
+    })
```

Comment:
This test makes sense, every load should be an animation.
So test now what happens when the animate is false, you know?

Reply:
Changed the test too! 👍